SPECS-EXTENDED/mbedtls/mbedtls.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+ "Signatures": {
+  "v2.28.3.tar.gz": "bdf7c5bbdc338da3edad89b2885d4f8668f9a6fffeba6ec17a60333e36dade6f"
```

Comment:
This should also be updated. Since the name on the blobstore is `mbedtls-2.28.3.tar.gz`, we should update this as well.
There is a nifty option in the toolkit to automatically do it `SRPM_FILE_SIGNATURE_HANDLING=update `

Also, I would request to verify the build once again after the linter.

Reply:
Sure.